src/components/GreekWordsDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import GreekWordInfo from './GreekWordInfo';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+import IconButton from '@mui/material/IconButton';
+import { useEffect } from 'react';
+
+
+type greekWord = {
+  greekWords: string,
+  englishWords: string,
+  morphology: string,
+  descriptions: string[],
+};
+
+let word1 : greekWord = {
+  greekWords: "asdkfjasdgjkb/asdfasdf",
```

Comment:
I guess this is placeholder text?

Reply:
yes, I am currently using it to test the design that will be used to show the data from gwt